.env

```diff
@@ -0,0 +1,12 @@
+WARDEN_ENV_NAME=magento2
+WARDEN_ENV_TYPE=magento2
+TRAEFIK_DOMAIN=magento2.test
+TRAEFIK_SUBDOMAIN=app
+
+WARDEN_BLACKFIRE=1
+BLACKFIRE_CLIENT_ID=4a914d1f-17ab-4c69-943f-68219643c8f0
```

Comment:
Why blackfire secrets are here?

Reply:
:man_facepalming: Thanks for being aware : )